hw/top_darjeeling/ip/ast/ast.core

```diff
@@ -26,7 +26,7 @@ filesets:
       - lowrisc:ip:lc_ctrl_pkg
       - lowrisc:ip:edn_pkg
       - lowrisc:ip:entropy_src_pkg
-      - lowrisc:ip_interfaces:alert_handler_reg
+      - lowrisc:ip_interfaces:alert_handler_pkg
```

Comment:
This is a top specific core file. Use the top_specific core instead of ip_interfaces?

Reply:
Whoops, I think I accidentally went the opposite direction on the latest push and created another core to tie things together. I totally agree with your statement and will switch this to a direct dependency.